index.html

```diff
@@ -21,7 +21,7 @@ <h2>About devanooga</h2>
 
 <section>
   <h2>Our resources</h2>
-  <p>Our main resource is a <a href="/slack/">Slack Team</a>, where technologists from all around the region come together to share ideas, troubleshoot their projects, learn new things, or just hang out. We keep a relaxed atmosphere in there, but we do expect you to heed our <a href="/code-of-conduct/">Code of Conduct</a> (CoC). If you have any questions about the CoC, feel free to <a href="/contact/">use our contact form</a> to get in touch. You can also ask us on Slack, either publicly in the #general channel, or privately to any of the admins.</p>
+  <p>Our main resource is a <a href="/slack/">Slack Team</a>, where technologists from all around the region come together to share ideas, troubleshoot their projects, learn new things, or just hang out. We keep a relaxed atmosphere in there, but we do expect you to heed our <a href="/code-of-conduct/">Code of Conduct</a> (CoC). If you have any questions about the CoC, feel free to <a href="/contact/">use our contact form</a> to get in touch. You can also ask us on Slack, either publicly in the #general channel, or privately to any of the admins. If Slack is down, you can find us on IRC in <a href="irc://chat.freenode.net/devanooga">#devanooga</a></p>
```

Comment:
The added sentence should end with a period.

Reply:
You're absolutely right